src/page-modules/departures/server/journey-planner/index.ts

```diff
@@ -194,6 +207,48 @@ export function createJourneyApi(
 
       return validated.data;
     },
+
+    async estimatedCalls(input) {
+      const result = await client.query<
+        QuayEstimatedCallsQuery,
+        QuayEstimatedCallsQueryVariables
+      >({
+        query: QuayEstimatedCallsDocument,
+        variables: {
+          id: input.quayId,
+          numberOfDepartures: 10,
```

Comment:
Should we get as much as 10 or is 5 enough?

Reply:
I changed it to 5. I think that's a better experience for the user 👍